README.md

```diff
@@ -1,6 +1,47 @@
-# API Elements
+# API Elements: JavaScript
 
-## Using Lerna
+[API Elements](https://apielements.org/) is a uniform interface for dealing
+with API Descriptions. This repository contains tooling for handling API
+Elements in JavaScript along with parsers and serializers for API Description
```

Comment:
```suggestion
Elements in JavaScript along with parsers and serializers for API description
```

Reply:
Hmm, API description languages mean the same as API description formats. I wouldn't argue about domain dictionary now, but I suggested formats above and we should be at least consistent.